extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -120,26 +124,32 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 	};
 
 	GoogleTag.prototype.addSlot = function (adElement) {
+		console.log('bogna', 'calling addSlot', adElement.getSlotName(), window.googletag.getSlots());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS2228)